README.md

```diff
@@ -17,7 +17,7 @@ This is the backend client of Bridge-In-Tech.
 Please read our [Contributing Guidelines](.github/CONTRIBUTING.md), [Code of Conduct](.github/CODE_OF_CONDUCT.md) and [Reporting Guidelines](.github/REPORTING_GUIDELINES.md) thoroughly.
 
 ## Setup
-To start contributing to the project, setup the backend environment on your local machine by following the instructions on the [BIT Development Environment Setup Instruction]( https://github.com/anitab-org/bridge-in-tech-backend/wiki/BIT-development-environment-setup) wiki page.
+To start contributing to the project, setup the backend environment on your local machine by following the instructions on the [BIT Development Environment Setup Instruction]( https://github.com/anitab-org/bridge-in-tech-backend/blob/develop/.github/ENV_SETUP_INSTRUCTION.md) wiki page.
```

Comment:
```suggestion
To start contributing to the project, setup the backend environment on your local machine by following the instructions on the [BIT Development Environment Setup Instruction](.github/ENV_SETUP_INSTRUCTION.md) wiki page.
```

you could make the link as a relative path link :)

Reply:
@isabelcosta Thanks for reviewing. I have made the suggested changes 😀.